dns/opnblock/src/opnsense/scripts/OPNsense/opnblock/opnblock.sh

```diff
@@ -0,0 +1,22 @@
+# !/bin/sh
+# opnblock, opnblock.sh
+
+# prep temp storage and conf file
+rm -r '/tmp/hosts.working' '/var/unbound/opnblock.conf'
```

Comment:
why `-r` - it is a file. It stands for recursive to delete directories and their contents.

Reply:
Typo! Good catch!